Makefile

```diff
@@ -1,7 +1,12 @@
+.PHONY: init test
 init:
-    pip install -r requirements.txt
-
+	pip install pipenv --upgrade
+	pipenv install --dev
 test:
-    py.test tests
+    pytest
 
-.PHONY: init test
\ No newline at end of file
+publish:
+	pip install 'twine>=1.5.0'

```

Comment:
Installing a package automatically might disrupt the user environment.
Just printing a version through `python -m twine --version` and check its exit code might be necessary.
Twine 1.5.0 is released 5 years ago, so it is not generally expected a user having a 5 years old version of twine. If you want to make sure a user to have >= 1.5.0, you can do this:
```
python3 -c 'from packaging import version; import twine; assert version.parse(twine.__version__) >= version.parse("1.5.0")'
```

Reply:
`publish` command is only for us. I think we don't need to consider other users.

Reply:
1. It is bad because it'll select python 2.7 in most environment.
2. If I forget to activate venv, it'll install twine under /usr/local/lib/python2.7 or, /usr/local/lib/python3.6. Linux will require sudo to install globally, but my Mac+ Homebrew doesn't require that. Automatically installs it. 
3. If my python get upgraded to 3.7 or 3.8, installed twine is still left /usr/local/lib/python3.6/...
System package manager usually doesn't care about python packages. It'll never get deleted until I delete manually.

Reply:
I changed it.

Reply:
I think just printing an error message if the user does not have the package will be enough.